AnyServe/AnyServe/internalLog.txt

```diff
@@ -0,0 +1,85 @@
+2020-10-03 16:09:10.4748 Info Message Template Auto Format enabled
+2020-10-03 16:09:10.5016 Info Loading assembly: NLog.Web.AspNetCore
+2020-10-03 16:09:10.5692 Info Adding target FileTarget(Name=alldata)
+2020-10-03 16:09:10.5854 Info Adding target FileTarget(Name=anyserv-web)
+2020-10-03 16:09:10.6228 Info Validating config: TargetNames=alldata, anyserv-web, ConfigItems=45, FilePath=/src/bin/netcoreapp3.1/nlog.config
+2020-10-03 17:23:23.4532 Info Message Template Auto Format enabled
+2020-10-03 17:23:23.4791 Info Loading assembly: NLog.Web.AspNetCore
+2020-10-03 17:23:23.5441 Info Adding target FileTarget(Name=alldata)
+2020-10-03 17:23:23.5597 Info Adding target FileTarget(Name=anyserv-web)
+2020-10-03 17:23:23.5972 Info Validating config: TargetNames=alldata, anyserv-web, ConfigItems=45, FilePath=/src/bin/netcoreapp3.1/nlog.config
+2020-10-03 17:32:15.5583 Info Message Template Auto Format enabled
+2020-10-03 17:32:15.5865 Info Loading assembly: NLog.Web.AspNetCore
+2020-10-03 17:32:15.6708 Info Adding target FileTarget(Name=alldata)
+2020-10-03 17:32:15.6878 Info Adding target FileTarget(Name=anyserv-web)
+2020-10-03 17:32:15.7301 Info Validating config: TargetNames=alldata, anyserv-web, ConfigItems=45, FilePath=/src/bin/netcoreapp3.1/nlog.config
+2020-10-03 17:38:40.0229 Info Message Template Auto Format enabled
+2020-10-03 17:38:40.0473 Info Loading assembly: NLog.Web.AspNetCore
+2020-10-03 17:38:40.1121 Info Adding target FileTarget(Name=alldata)
+2020-10-03 17:38:40.1330 Info Adding target FileTarget(Name=anyserv-web)
+2020-10-03 17:38:40.1811 Info Validating config: TargetNames=alldata, anyserv-web, ConfigItems=45, FilePath=/src/bin/netcoreapp3.1/nlog.config
+2020-10-03 17:43:03.1476 Info Message Template Auto Format enabled
+2020-10-03 17:43:03.1767 Info Loading assembly: NLog.Web.AspNetCore
+2020-10-03 17:43:03.2513 Info Adding target FileTarget(Name=alldata)
+2020-10-03 17:43:03.2650 Info Adding target FileTarget(Name=anyserv-web)
+2020-10-03 17:43:03.3026 Info Validating config: TargetNames=alldata, anyserv-web, ConfigItems=45, FilePath=/src/bin/netcoreapp3.1/nlog.config
+2020-10-03 17:50:03.4582 Info Message Template Auto Format enabled
+2020-10-03 17:50:03.4830 Info Loading assembly: NLog.Web.AspNetCore
+2020-10-03 17:50:03.5534 Info Adding target FileTarget(Name=alldata)
+2020-10-03 17:50:03.5680 Info Adding target FileTarget(Name=anyserv-web)
+2020-10-03 17:50:03.6053 Info Validating config: TargetNames=alldata, anyserv-web, ConfigItems=45, FilePath=/src/bin/netcoreapp3.1/nlog.config
+2020-10-03 17:54:54.7657 Info Message Template Auto Format enabled
+2020-10-03 17:54:54.8000 Info Loading assembly: NLog.Web.AspNetCore
+2020-10-03 17:54:55.0107 Info Adding target FileTarget(Name=alldata)
+2020-10-03 17:54:55.0759 Info Adding target FileTarget(Name=anyserv-web)
+2020-10-03 17:54:55.1279 Info Validating config: TargetNames=alldata, anyserv-web, ConfigItems=45, FilePath=/src/bin/netcoreapp3.1/nlog.config
```

Comment:
Don't add to master log files please

Reply:
I don't have any idea what you talked about